micrometer-commons/src/main/java/io/micrometer/common/docs/KeyName.java

```diff
@@ -46,6 +46,14 @@ default KeyValue withValue(String value) {
         return KeyValue.of(this, value);
     }
 
+    /**
+     * Creates a key value for the given key name with {@value KeyValue#NONE_VALUE}.
+     * @return key value
+     */
+    default KeyValue withoutValue() {
```

Comment:
I'm thinking about the name of this, what do you think about `withNoneValue`?
Not sure if it's better though and maybe `withoutValue` is the more user-friendly choice.
/cc @shakuzen

Reply:
@jonatan-ivanov Thanks for the feedback!

Actually, I considered it initially, and I'm okay with either way, so just let me know which one you prefer.

Reply:
I think `withNoneValue` is more clear. `withoutValue` to me sounds like a KeyValue with a `null` or otherwise missing value, which isn't possible, but that's what it sounds like it means.

Reply:
@shakuzen Thanks for the feedback!

I renamed it in a4e4bfe.